### DISCUSSION TITLE
How will the Root Cert get renewed in Cluster?

### DISCUSSION THREAD

**Discussion by cloudscape-germany** [kube-hetzner/terraform-hcloud-kube-hetzner#1511] [Q&A]
Root Cert between etcd, mater and worker nodes usually expires after one year and last time using Hetzner and Kubermatic I got locked out and lost the entire company, my client and the contract. Is there anything better with Suse and their Cluster Root Cert after one year what happens? Anyone ran into this?

**Comment by rmelilloii** [comment]
Hello and good morning! I have been using "hetzner-k3s" to deploy my clusters on ubuntu. I never had this problem there, is this something I should worry about with this solution? 

Thanks!
